### ISSUE TITLE
[maybe] Allow "break ..." after control structures

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#484]
Currently `Squiz.WhiteSpace.ControlStructureSpacing` sniff, that there should be an empty line after control structure, e.g.
```php
if ( true ) {
    // ...
}

```
I've recently stumbled upon this case:
```php
foreach($array as $value) {
    // ...
    if ( $value ) {
        if ( $somethingElse ) {
            // ...
        }
        break;
    }
}
```

So we have nested IF's within a foreach and `break` or `break 2` will break from `foreach`. Question is whatever it should be allowed to have `break` without empty line between it and previous IF (or control structure in general).

P.S.
I know, that in `switch` we allow for such case (no empty line before `break`, when `case/default` ends up with control structure.

**Comment by gsherwood** [comment]
> Question is whatever it should be allowed to have break without empty line between it and previous IF (or control structure in general).

I don't want that sniff to allow that code.
